schema.sql

```diff
@@ -1,5 +1,27 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int NOT NULL  PRIMARY KEY,
+	name VARCHAR(100),
+	date_of_birth date NOT NULL,
+	escape_attempts integer,
+	neutered boolean NOT NULL,
+	weight_kg numeric NOT NULL,
+	species VARCHAR(50),
+);
+
```

Comment:
- [ ] Here should write queries that modify the `animals` table. Kindly take a look at the screenshot from the project requirements.
![Screenshot from 2022-01-14 20-11-44](https://user-images.githubusercontent.com/20245164/149557162-cbed4c0a-3d95-4da1-9b8a-e99d37a92b45.png)

Check the sample code:
```sql
 ALTER TABLE REMOVE species;
 ALTER TABLE animals ADD species_id int REFERENCES species(id);
 ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```

Reply:
Thanks a lot